src/content/cloud/namespaces.mdx

```diff
@@ -7,13 +7,13 @@ id: namespaces
 
 import Image from '@theme/Image';
 
-With Okteto Cloud you can create and manage secure and isolated Kubernetes namespaces for you and your team.
+With Okteto dashboardu can create and manage secure and isolated Kubernetes namespaces for you and your team.
 
-Okteto Cloud will take care of all the required RBAC roles and bindings, resources quotas and limits, pod security policies, and network policies so everyone in your team can safely deploy and develop applications in Kubernetes.
+Okteto dashboardll take care of all the required RBAC roles and bindings, resources quotas and limits, pod security policies, and network policies so everyone in your team can safely deploy and develop applications in Kubernetes.
```

Comment:
```suggestion
The Okteto platform will take care of all the required RBAC roles and bindings, resources quotas and limits, pod security policies, and network policies so everyone in your team can safely deploy and develop applications in Kubernetes.
```
"platform" seems more apt here (vs "dashboard") because we're referring to the underlying magic vs the UI/web app specifically (I think). Non-blocking suggestion